.github/CODEOWNERS

```
@@ -95,19 +76,19 @@ xpti/ @tovinkere @andykaylor
 xptifw/ @tovinkere  @andykaylor
 
 # DPC++ tools
-llvm/**/file-table-tform/ @kbobrovs @AlexeySachkov
+llvm/**/file-table-tform/ @mlychkov @AlexeySachkov
 llvm/**/llvm-foreach/ @AlexeySachkov  @Fznamznon
 llvm/**/llvm-no-spir-kernel/ @AGindinson @AlexeySachkov
-llvm/**/sycl-post-link/ @kbobrovs @AlexeySachkov
-llvm/include/llvm/Support/PropertySetIO.h @kbobrovs @AlexeySachkov
-llvm/lib/Support/PropertySetIO.cpp @kbobrovs @AlexeySachkov
-llvm/unittests/Support/PropertySetIOTest.cpp @kbobrovs @AlexeySachkov
-llvm/lib/Support/Base64.cpp @kbobrovs @AlexeySachkov
+llvm/**/sycl-post-link/ @mlychkov @AlexeySachkov
+llvm/include/llvm/Support/PropertySetIO.h @mlychkov @AlexeySachkov
+llvm/lib/Support/PropertySetIO.cpp @mlychkov @AlexeySachkov
+llvm/unittests/Support/PropertySetIOTest.cpp @mlychkov @AlexeySachkov
+llvm/lib/Support/Base64.cpp @mlychkov @AlexeySachkov
 
 # Clang offload tools
-clang/tools/clang-offload-bundler/ @kbobrovs @sndmitriev
-clang/tools/clang-offload-wrapper/ @sndmitriev @kbobrovs
-clang/tools/clang-offload-deps/ @sndmitriev
+clang/tools/clang-offload-bundler/ @mlychkov @sndmitriev
+clang/tools/clang-offload-wrapper/ @mlychkov @sndmitriev
```

@AlexeySachkov, do you want to be added for bundler and wrapper tools as well?
```suggestion
clang/tools/clang-offload-bundler/ @mlychkov @sndmitriev @AlexeySachkov
clang/tools/clang-offload-wrapper/ @mlychkov @sndmitriev @AlexeySachkov
```

---

Yes, I think I can be added as a third owner to all `clang-offload-` tools we have